grafana-plugin/src/containers/MobileAppConnection/MobileAppConnection.tsx

```diff
@@ -192,12 +192,19 @@ export const MobileAppConnection = observer(({ userPk }: Props) => {
     );
   }
 
+  const isMobile = window.matchMedia('(max-width: 768px)').matches;
```

Comment:
can we put it in some global place like `src/utils/utils`?

Reply:
done ✅